src/test/groovy/ee/tuleva/onboarding/mandate/GenericMandateIntegrationTest.java

```diff
@@ -65,7 +65,8 @@ static Stream<Arguments> testMandateDetails() {
                 SECOND,
                 FundPensionOpeningMandateDetails.FundPensionFrequency.MONTHLY,
                 new FundPensionOpeningMandateDetails.FundPensionDuration(20, false),
-                new BankAccountDetails(ESTONIAN, BankAccountDetails.Bank.LHV, "EE_TEST_IBAN"))));
+                new BankAccountDetails(ESTONIAN, BankAccountDetails.Bank.LHV, "EE_TEST_IBAN"))),
```

Comment:
You can use a static import for the bank

Reply:
Should see if there is a spotless setting for it